scripts/travis/pylint_harness.sh

```diff
@@ -15,11 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-disabled="--disable=similarities,invalid-name,too-many-statements,too-many-arguments,too-many-locals,too-few-public-methods,relative-import,no-self-use"
+disabled="--disable=raise-missing-from,line-too-long,wrong-import-order,similarities,invalid-name,too-many-statements,too-many-arguments,too-many-locals,too-few-public-methods,relative-import,no-self-use"
 
 pylint ${disabled} --rcfile pylintrc ensembl_genes/ > output.err
 
-grep -v "\-\-\-\-\-\-\-\-\-" output.err | grep -v "Your code has been rated" | grep -v "\n\n" | sed '/^$/d' > pylint.err
+grep -v "\-\-\-\-\-\-\-\-\-" output.err | grep -v "\*\*\*\*\*\*\*\*\*\*\*\*\*" | grep -v "Your code has been rated" | grep -v "\n\n" | sed '/^$/d' > pylint.err
```

Comment:
There is an option for pylint for the output, `--msg-template=<format string>`. Maybe it can be used instead of multiple greps

Reply:
I've had a look and I couldn't find how to use that parameter to remove the lines we want but maybe we can change it in future if we learn how to do it.